### ISSUE TITLE
Where is search button?

### ISSUE THREAD

**Issue by Nodeist** [paritytech/substrate-telemetry#473]
...

**Comment by jsdw** [comment]
If you start typing, you'll filter on the list of nodes that's displayed!

I would be in favour of making this a little more explicit by adding an icon or something which also opens this filter/search box.

**Comment by Nodeist** [comment]
definitely. It's not possible to do what you're saying on a mobile device.

**Comment by jsdw** [comment]
THat's a fair point!
